recipes/physx/4.1.1/conanfile.py

```diff
@@ -0,0 +1,266 @@
+import os
+import time
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+class PhysXConan(ConanFile):
+    name = "physx"
+    description = "The NVIDIA PhysX SDK is a scalable multi-platform " \
+                  "physics solution supporting a wide range of devices, " \
+                  "from smartphones to high-end multicore CPUs and GPUs."
+    license = "BSD-3-Clause"
+    topics = ("conan", "PhysX", "physics")
+    homepage = "https://github.com/NVIDIAGameWorks/PhysX"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake"
+    settings = "os", "compiler", "arch", "build_type"
+    short_paths = True
```

Comment:
Is this really required? I'm just asking.

Reply:
Mmhhh, I don't remember. Maybe I've forgotten to remove this line when I created the recipe from an other one. I'll check.

Reply:
I confirm that it's required:

```
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(186,5): error MSB4018: System.InvalidOperationException: The item metadata "%(FullPath)" cannot be applied to the path "LowLevelDynamics.dir\release\LowLevelDynamics.tlog\ParallelCustomBuild.command.1.tlog". Path: LowLevelDynamics.dir\release\LowLevelDynamics.tlog\ParallelCustomBuild.command.1.tlog exceeds the OS max path limit. The fully qualified file name must be less than 260 characters. [C:\Users\appveyor\.conan\data\physx\4.1.1\SpaceIm\testing\build\39d7f8847ad0422581979b1100f58fd141e8e33a\build_subfolder\source_subfolder\physx\compiler\public\sdk_source_bin\LowLevelDynamics.vcxproj]
```

Reply:
Thanks! Too bad..